routes/helperMethods.js

```diff
@@ -10,6 +10,7 @@ function returnData(data, res) {
 
 //set status code of the response and send error info to the user in json
 function errorHandler(res, status, err) {
+  if (res.headersSent) return; 
```

Comment:
The error we were getting was `[ERR_HTTP_HEADERS_SENT: Cannot set headers after they are sent to the client](https://stackoverflow.com/questions/52122272/err-http-headers-sent-cannot-set-headers-after-they-are-sent-to-the-client)`. We believe that it was an issue of multiple responses getting sent for ONE request, either from returnData on line 131 or errorHandler on lines 134 and 164. So this line of code ensures that we send one response per request, even in the event that errorHandler is called multiple times.